content/300-guides/200-deployment/301-edge/550-deploy-to-deno-deploy.mdx

```diff
@@ -84,44 +84,41 @@ With the data model in place and your database connection configured, you can no
 ### Steps
 
 ```terminal
-deno run -A npm:prisma db push
+deno run -A npm:prisma migrate dev --name init
```

Comment:
Why the switch from push to migrate dev here?

Reply:
Since we're working with a PostgreSQL database, we should recommend "best practice", which is using `prisma migrate dev` to track migrations.

Reply:
Why did we not do that before?

Reply:
No idea. 😄